ntoskrnl/io/iomgr/deviface.c

```diff
@@ -412,6 +686,175 @@ IopOpenInterfaceKey(IN CONST GUID *InterfaceClassGuid,
     return Status;
 }
 
+/**
+ * @brief
+ * Returns the alias device interface of the specified device interface
+ * instance, if the alias exists.
+ *
+ * @param[in] SymbolicLinkName
+ * Pointer to a symbolic link string which identifies the device interface instance.
+ *
+ * @param[in] AliasInterfaceClassGuid
+ * Pointer to a device interface class GUID.
+ *
+ * @param[out] AliasSymbolicLinkName
+ * Pointer to unicode string which receives the alias symbolic link upon success.
+ * Must be freed with RtlFreeUnicodeString after using.
+ *
+ * @return NTSTATUS values in case of errors, STATUS_SUCCESS otherwise.
+ *
+ * @remarks Must be called at IRQL = PASSIVE_LEVEL in the context of a system thread
+ */
+NTSTATUS
+NTAPI
+IoGetDeviceInterfaceAlias(
+    _In_ PUNICODE_STRING SymbolicLinkName,
+    _In_ CONST GUID *AliasInterfaceClassGuid,
+    _Out_ PUNICODE_STRING AliasSymbolicLinkName)
+{
+    UNICODE_STRING AliasSymbolicLink = {0};
+    UNICODE_STRING PrefixString = {0};
+    UNICODE_STRING AliasGuidString = {0};
+    UNICODE_STRING DeviceString = {0};
+    UNICODE_STRING ReferenceString = {0};
+    PKEY_VALUE_FULL_INFORMATION kvInfo;
+    HANDLE DeviceKey, AliasInstanceKey;
+    PVOID Buffer;
+    USHORT i;
+    NTSTATUS Status;
+
+    DPRINT("IoGetDeviceInterfaceAlias(): symbolic link '%wZ'\n", SymbolicLinkName);
```

Comment:
```suggestion
    DPRINT("IoGetDeviceInterfaceAlias('%wZ', 0x%p)\n", SymbolicLinkName, AliasInterfaceClassGuid);
```
but I don't know whether we have to keep this trace as well.